### PR TITLE
430 use `expect_no_error` in tests instead of `expect_error(, NA)`

### PR DESCRIPTION
**Pull request by m7pr**
Part of https://github.com/insightsengineering/coredev-tasks/issues/430

### CODE REVIEW

File: tests/testthat/test-module_nested_tabs.R [hunk 1]
```diff
@@ -148,29 +148,29 @@ out <- shiny::testServer(
   ),
   expr = {
     testthat::test_that("modules_reactive is a list of reactives", {
-      expect_is(modules_reactive, "list")
-      expect_is(modules_reactive$tab1, "reactive")
-      expect_is(modules_reactive$tab2, "reactive")
+      testthat::expect_is(modules_reactive, "list")
+      testthat::expect_is(modules_reactive$tab1, "reactive")
+      testthat::expect_is(modules_reactive$tab2, "reactive")
```
Comment (averissimo) [inline tests/testthat/test-module_nested_tabs.R:153]: Added some `testthat::` missing prefixes. Let me know if I should revert this change or resolve and merge :grin:

**Comment by m7pr** [comment]
I think this is fine. Thanks @averissimo
